rust/crd/src/tls.rs

```diff
@@ -0,0 +1,26 @@
+use serde::{Deserialize, Serialize};
+use stackable_operator::schemars::{self, JsonSchema};
+
+#[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NifiTls {
+    /// Only affects client connections.
+    /// This setting controls:
+    /// - Which cert the servers should use to authenticate themselves against the client
```

Comment:
The list of one item seems redundant.

Also, is this statement true? 

> ... to authenticate themselves against the client

```suggestion
    /// This only affects client connections and is used to
    /// control which certificate the servers should use to
    /// authenticate themselves against the client.
```

Reply:
I know the list is redundant, I just wanted to be consistent with other operators, such as https://github.com/stackabletech/trino-operator/blob/9d5b2145efd3938790dd2ff532c1437d282d6ef0/rust/crd/src/lib.rs#L278.

But I guess the typical user wont notice, so just using you suggestion